src/inc_gauged_raz.rs

```diff
@@ -37,6 +37,7 @@ const DEFAULT_SECTION_CAPACITY: usize = 500;
 /// The data stored in the tree structure of the RAZ.
 #[derive(PartialEq,Eq,Debug,Hash,Clone)]
 enum TreeData<E:Debug+Clone+Eq+Hash, M:RazMeta<E>> {
+	Dummy, // used when rebuild is certain
```

Comment:
Is `Dummy` just a micro optimization? (If so, that may be fine).
That is, I'm wondering what's wrong with using `Leaf(Rc::new(vec![]))` in all places below where you use `Dummy`?